docs/extras/modules/data_connection/vectorstores/integrations/convex.mdx

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_class_name: node-only
+---
+
+# Convex
+
+LangChain.js supports [Convex](https://convex.dev/) as a [vector store](https://docs.convex.dev/vector-search), and supports the standard similarity search.
+
+## Setup
+
+### Create project
+
+Get a working [Convex](https://docs.convex.dev/) project set up, for example by using:
+
+```bash
+npm create convex@latest
+```
+
+### Add database accessors
+
+Add query and mutation helpers to `convex/langchain/db.ts`:
+
+```ts title="convex/langchain/db.ts"
+export * from "langchain/util/convex";
+```
+
+### Configure your schema
+
```

Comment:
It seems like `npx convex dev` is also a required step?

Reply:
`npm create convex` includes directions for `npm run dev`, which calls `npx convex dev` under the hood